src/components/tab-nav/tab-nav.tsx

```diff
@@ -3,6 +3,7 @@ import {
   Element,
   Event,
   EventEmitter,
+  forceUpdate,
```

Comment:
Instead of calling forceUpdate, is there an internal prop that could be added on the element or something in order to more naturally trigger a rerender?

Reply:
It fails to update `tabs` in a few places when `tabTitle` changes due to the selection moving after `tab-title` closes. I'll be following this up with a refactor issue on all tab-related components, as they are a bit more complicated than might need to be. And to address inconsistent variable naming. I'll add `forceUpdate` as an item to address.